### ISSUE TITLE
Update CPAN version of this module?

### ISSUE THREAD

**Issue by gwselke** [DBD-SQLite/DBD-SQLite#68]
Thank you for this very useful module! Could you, please, occasionally update also the CPAN version of this module which is, as far as I can see, still at v1.66 of August 2020, using a somewhat dated und *much less* efficient version of sqlite? It would be great for the technically less involved users to be able to profit from the newer version.
Thank you!

\Gisbert

**Comment by tobyink** [comment]
+1 this would be great

**Comment by gwselke** [comment]
By all means! No need to hurry. Thank you for this great module! 
-- 
\Gisbert

**Comment by hbarnard** [comment]
I'm getting this currently, using cpanm. Going to try manual install with tarball now.
```
{standard input}: Assembler messages:
{standard input}:1688776: Warning: end of file not at end of a line; newline inserted
x86_64-linux-gnu-gcc: fatal error: Killed signal terminated program cc1
compilation terminated.
make: *** [Makefile:367: sqlite3.o] Error 1
-> FAIL Installing DBD::SQLite failed.
```

**Comment by charsbar** [comment]
Shipped 1.68. Thanks.

**Comment by gwselke** [comment]
Thank you! This is great! 
-- 
\Gisbert

Am 22. Juli 2021 07:30:05 MESZ schrieb Kenichi Ishigaki ***@***.***>:
